### PR TITLE
Improve Notification Bar

### LINKED ISSUE #5: Notification bar improvements
For the notification bar that can be enabled by setting params.notifcation.enable to true in config.toml
- [ ] Make it closable
- [ ] When at top, don't cover menus but float (stay at top of screen) on scroll

### PR DESCRIPTION
**Pull request by hanzei**
- [x] Make it closable
- [x] When at top, don't cover menus but float (stay at top of screen) on scroll

|  After |  Before |
|---|---|
| screenshot from 2018-11-28 16-14-45  |  screenshot from 2018-12-04 09-15-10  |

There a slight changes to the background. I'm not an CSS expert. For comparison  with a closed notification:

|  After |  Before |
|---|---|
| screenshot from 2018-11-28 16-11-13 | screenshot from 2018-11-28 16-11-18 | 

Fixes #5

### CODE REVIEW

File: site/static/css/styles.css [hunk 1]
```diff
@@ -3445,9 +3445,11 @@ table>tbody>tr:nth-of-type(odd) {
 .homepage header {
     background: transparent;
     color: #fff;
+    /*
     left: 0;
     position: absolute;
     top: 0;
+    */
```
Comment (jwilander) [inline site/static/css/styles.css]: Left in on purpose or should we remove?

**Comment by jwilander** [comment]
@jasonblais you might want to take a look. Note that the colour didn't change. We can change that to be more appropriate when we actually use the notification bar for something

File: site/static/css/styles.css (hunk 1 above)
Comment (hanzei) [reply site/static/css/styles.css]: This should be removed. Fixed.

**Comment by jasonblais** [comment]
@hanzei @jwilander It's not immediately clear what the improvement is? (I couldn't decipher it from the diff file).

**Comment by hanzei** [comment]
@jasonblais I updated the screenshots to better represent the changes. Please take a look.

**Comment by jasonblais** [comment]
@hanzei Thank you! This helps a lot.

Can you help make the whitespace left and right of the text equal? Currently it's greater on the left hand side.

image


A designer suggested 32px on both sides, but I'm not sure if that's the appropriate value.

**Comment by hanzei** [comment]
I changed it so the text has 32px spacing to the left edge and the close icon right. Is this what you meant?

screenshot from 2018-12-06 09-31-46

I looks unproportional to me because the text leans to the left side. I would rather have a equal spacing to the left and right edge.

**Comment by jasonblais** [comment]
@hanzei Hmm, that feels wrong, I agree. 

I hate to do this, but can you help revert back to what you had and we'll merge it instead? I'm looking at the notification bar we use for the Mattermost server and the spacing matches with what you had before.

image

I really really apologize for making you go back and forth, I'll do a more careful job next time before suggesting a change :)

**Comment by hanzei** [comment]
Sure thing, no problem. Reverting a commit isn't a big deal. :wink:

**Comment by jasonblais** [comment]
> Sure thing, no problem. Reverting a commit isn't a big deal. 😉

Fair enough :)

**Pull request merged**